README.md

```diff
@@ -28,49 +28,49 @@ We use `main` branch as the develop branch while setting the default branch as t
+| AAD               | [spring-cloud-azure-starter-active-directory:4.4.0]          | [web-client-access-resource-server](aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server)                                                                                                                                         |
+| AAD               | [spring-cloud-azure-starter-active-directory:4.4.0]          | [aad-webapp-resource-server](aad/spring-cloud-azure-starter-active-directory/aad-web-application-and-resource-server)                                                                                                                                          |
 | App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-sample](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-sample)                                                                                             |
 | App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-convert-sample-complete](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-convert-sample/azure-spring-cloud-appconfiguration-config-convert-sample-complete) |
 | App Configuration | [azure-spring-cloud-appconfiguration-config:2.8.0]           | [azure-spring-cloud-appconfiguration-config-convert-sample-initial](appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-convert-sample/azure-spring-cloud-appconfiguration-config-convert-sample-initial)   |
 | App Configuration | [azure-spring-cloud-feature-management:2.7.0]                | [azure-spring-cloud-feature-management-sample](appconfiguration/azure-spring-cloud-feature-management/azure-spring-cloud-feature-management-sample)                                                                                                            |
 | App Configuration | [azure-spring-cloud-feature-management-web:2.7.0]            | [azure-spring-cloud-feature-management-web-sample](appconfiguration/azure-spring-cloud-feature-management-web/azure-spring-cloud-feature-management-web-sample)                                                                                                |
 | App Configuration | [azure-spring-cloud-starter-appconfiguration-config:2.8.0]   | [azure-spring-cloud-starter-appconfiguration-config-sample](appconfiguration/azure-spring-cloud-starter-appconfiguration-config/azure-spring-cloud-starter-appconfiguration-config-sample)                                                                     |
```

Comment:
app configure related version should be updated too. https://github.com/Azure-Samples/azure-spring-boot-samples/pull/503/files